### ISSUE TITLE
PsychoPy 2023.1.1 still has BUG reported in #4207 

### ISSUE THREAD

**Issue by drkitty28** [psychopy/psychopy#5514]
I've recently installed PsychoPy 2023.1.1 in a venv using Python 3.8 on Ubuntu 22.04 and get exactly the same problem described in #4207.  I've tried the workaround of revising the paramCtrls.py.org file and still have the problem.  I followed the related posts, but didn't see a resolution that worked for me.  Is this still a bug?  Is there another workaround?  Thanks.  
BTW, strangely,  this only applies to the text component. For any other component (sound, image, etc) using the $ does not crash PsychoPy

![psychopyError](https://user-images.githubusercontent.com/26172078/231456037-77ee547c-1e3a-4623-849f-7d3d603d2d0e.png)

**Comment by TEParsons** [comment]
It looks like the version you're actually running is 2022.2.5 - the line in the error message has moved from position 550 since then, if you were in 2023.1.1 it would be line 1005. Did you install via the button on psychopy.org or via the file [here](https://github.com/psychopy/psychopy/releases/tag/2023.1.1)? What version number is listed on the top bar of the app when you open it?

**Comment by drkitty28** [comment]
I used the 2023.1.1 version from PsychoPy releases - looks like it's now 2023.1.2 as of 11 hrs ago from the link.  Created a venv, set the Python version to 3.8, unzipped and used the setup.py with build and install.  All went well, and in fact, most of the demos run and my old scripts run, but if I create a new script (Builder) or try to modify an old script, I get the $var problem.  Weirdly, only in the text component.  The sound and image components do not crash PsychoPy when using $var.  You're right, though, what launches from the PsychoPy-2023.1.1.zip file is 2022.2.4 - even though it unzips as a 2023.1.1 directory.  Didn't notice that until you mentioned it.  But, I did notice that the PsychoPy-2023.1.1.tar.gz file produced the wrong (earlier) version which is why I downloaded the zip.

![image](https://user-images.githubusercontent.com/26172078/231745762-ef123fe9-3dc7-4f8b-8fec-f6d2161ff14e.png)

![image](https://user-images.githubusercontent.com/26172078/231744989-5632efcd-bc9e-43ab-bf8b-2af8a7029979.png)

![image](https://user-images.githubusercontent.com/26172078/231600935-de37c422-b709-45c8-b0ec-84b9134ad25c.png)

![image](https://user-images.githubusercontent.com/26172078/231598578-92f914c1-d2ee-45f5-88c2-26f4f4d43df5.png)

**Comment by peircej** [comment]
I've just unzipped the released files from https://github.com/psychopy/psychopy/releases/tag/2023.1.1 to see if they accidentally had the wrong files inside but, no, they correctly refer to their versions as 23.1.1 not 22.2.4 so my best guess is that your system is somehow loading the wrong psychopy (e.g. you've got some older files on your path that's being picked up). Could you go to the terminal in the app and check:
```
import psychopy
print(psychopy)  # to check the path of the lib
print(psychopy.__version__)
```

**Comment by peircej** [comment]
I now see you have another thread discussing the same bug and in that one you seem to have solved the issues of running the wrong version. I'm closing in favour of #5518 so we don't have multiple nearly-identical threads

**Comment by seymatakir** [comment]
I'm having the same problem, did you find any solution?
Ubuntu:20.04, python:3.8

**Comment by TEParsons** [comment]
If you can run PsychoPy directly from Python (e.g. through VS Code, PyCharm, etc.) then you should be able to run the current `dev` branch which includes a fix for this, otherwise you may be best off waiting for the next major release (which will be early 2024) which will also include the fix.

**Comment by peircej** [comment]
Just to reiterate, I think this _is_ already released in 2023.2
